src/components/Sidebar/Navbar.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+
+import MenuNavLink from './MenuNavLink';
+import { ReactComponent as ChevronDownSVG } from '../../assets/images/chevron-down.svg';
+import { ReactComponent as ChevronUpSVG } from '../../assets/images/chevron-up.svg';
+import { ReactComponent as HomeSVG } from '../../assets/images/home.svg';
+import { ReactComponent as MenuSVG } from '../../assets/images/menu.svg';
+import './sidebar.css';
+import machineLearningExamples from '../../data/Example';
+import { SidebarProps } from '../../types/types';
+import SVGIcon from '../SVGIcon/SVGIcon';
+
+const Sidebar: React.FC<SidebarProps> = ({ showMobileSidebar }: SidebarProps) => {
+  const [showMachineLearningLinks, setShowMachineLearningLinks] = useState<boolean>(false);
+
+  const handleMachineLearningClick = (): void => {
+    setShowMachineLearningLinks(!showMachineLearningLinks);
+  };
+
+  return (
+    <nav className={showMobileSidebar ? 'sidebar sidebar--mobile' : 'sidebar'}>
+      <MenuNavLink
+        to='/'
+        desc={
+          <>
+            <SVGIcon Icon={HomeSVG} />
+            <p>Start</p>
+          </>
+        }
+      />
+
+      <section className='machine-learning'>
+        <section
+          role='button'
+          tabIndex={0}
+          className='machine-learning__header'
+          onClick={handleMachineLearningClick}
+          onKeyDown={(e) => {
```

Comment:
You should avoid using lamba functions directly here in callback. You should define these callback as constants wrapping them with 'useCallback' hook to cache them. Or even declare them out of the component. You do not want that browser declare them and initialize each time component re-renders, it's time consuming. Check this article:

https://jonhilton.net/typescript-and-react-forbidden-lambdas/